mps-legacy-sync-plugin/src/main/java/org/modelix/model/mpsplugin/plugin/CloudBranchGroupActionGroup.kt

```diff
@@ -0,0 +1,19 @@
+package org.modelix.model.mpsplugin.plugin
+
+import jetbrains.mps.plugins.actions.GeneratedActionGroup
+import jetbrains.mps.workbench.action.ApplicationPlugin
+
+/*Generated by MPS */
+class CloudBranchGroupActionGroup(plugin: ApplicationPlugin) : GeneratedActionGroup("CloudBranchGroup", ID, plugin) {
+    init {
+        setIsInternal(false)
+        isPopup = false
+        this@CloudBranchGroupActionGroup.addAction("org.modelix.model.mpsplugin.plugin.AddBranch_Action")
```

Comment:
The class was renamed, but here the old name is still used. Maybe it's easier to disable this ktlint rule for now.